documentation/asciidoc/topics/con_fips_mode_cr.adoc

```diff
@@ -0,0 +1,20 @@
+[id="fips-mode-CR_{context}"]
+= FIPS mode for your `Infinispan` CR
+
+[role="_abstract"]
+The {openshiftplatform} can use certain Federal Information Processing Standards (FIPS) components that ensure {openshiftshort} clusters meet the requirements of a FIPS compliance audit.
+
+If you enabled FIPS mode on your {openshiftshort} cluster then the {ispn_operator} automatically enables FIPS mode for your `{brandname}` CR.
+
+[IMPORTANT]
+====
+Client certificate authentication is not currently supported with FIPS mode. Attempts to create {brandname} CR with `spec.security.endpointEncryption.clientCert` set to a value other than `None` will fail.
+====
+
+[role="_additional-resources"]
+.Additional resources
+* link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html/installing/installing-fips#doc-wrapper[Support for FIPS cryptography {openshiftplatform}]
+* link:https://access.redhat.com/documentation/en-us/red_hat_data_grid/{ispn_version}}/html-single/data_grid_operator_guide/index#setting-jvm-options_services[Setting JVM options]
```

Comment:
I don't think we need this link any more.

Reply:
I removed the "Setting JVM options" link.